<h2>{{message}}</h2>
<div class="information">
	<p>In prokaryotic organisms, a substantial fraction of adjacent genes are related functionally and most of these are organized into operons.  Operons are a group of co-directionally organized genes in prokaryotic genomes with the presence of a common promoter and terminator. Operons provide insight into the cellular functions and also help in determining distinct experimental designs. It is now clear that the structure of an operon is not static but rather changes with environmental conditions in bacterial species.</p>
	<p>We present OperomeDB, which provides an ensemble of all the predicted operons for bacterial genomes using available RNA-sequencing datasets across a wide-range of experimental conditions. Although several studies have recently confirmed that prokaryotic operon structure is dynamic with significant alterations across environmental and experimental conditions, there are no comprehensive databases for studying such variations across prokaryotic transcriptomes. To address this gap, we exploited the growing number of publicly available RNA-sequencing datasets from NCBI-SRA for various experimental conditions across diverse bacterial genomes, to provide a one stop portal for understanding the genome organization in the context of transcriptional regulation in a condition-specific manner. Currently our database contains nine bacterial organisms and 168 transcriptomes for which we predicted operons. User interface is simple and easy to use, in terms of visualization, downloading and querying of data. Users can browse through the reference genome, genes and operons predicted in the genome based on RNA-seq datasets as well as those identified in specific conditions. In addition, because of its ability to load custom datasets, users can also compare their datasets with publicly available transcriptomic data of an organism.</p>
</div>